SixTrack/sixtrack.s

```diff
@@ -18428,7 +18428,7 @@ cc2008
       write(4,*) ch
       goto 3
  90   continue
-10000 format(a16,1x,i2,1x,d21.15,1x,d21.15,1x,d16.10)
+10000 format(a16,1x,i2,1x,d22.15,1x,d22.15,1x,d17.10)
```

Comment:
This (in `subroutine write4`) changes `fort.4` -- "Geometry and strength Parameters (format as file # 2)". Or almost as file 2 -- it's missing the beam beam stuff in the single elements lists. Now it typically looks something like this:
```
 SINGLE

ip3               0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
ip3..2            0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
ip3..1            0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
d_c6t_649         0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
mqwa.a4r3.b1..1   0 0.964477668000000D-03 0.000000000000000D+00 0.0000000000D+00
q_c6t_1           0 0.100000000000000D+01 0.000000000000000D+00 0.0000000000D+00
drift_602         0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
drift_604         0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
mqwb.4r3.b1..1    0 -.535130001000000D-03 0.000000000000000D+00 0.0000000000D+00
drift_605         0 0.000000000000000D+00 0.000000000000000D+00 0.0000000000D+00
```

@rdemaria -- do you see any problem here? Any picky tooks that uses this file?

Also, why not just make it all `d22.15`?